CHANGELOG.md

```diff
@@ -8,9 +8,11 @@ To get the diff between two versions, go to https://github.com/FriendsOfECMAScri
 - 0.8.0
   - React new Context API is now used to manage state  
   - Decoupled from `react-intl` and `react-redux`
-  - Added React component testing using `react-testing-library`
   - [BC BREAK] Removed bridges for redux, browser, in-memory and local storage.
   - [BC BREAK] Removed support for React and ReactDOM < 16.3 (Context API is not supported in those versions)
+  - Added React component testing using `react-testing-library`
+  - Increased the test code coverage
+  - Upgraded all dependencies removing alpha and beta versions
```

Comment:
v0.8.0 was already published.

This should go in v0.8.1

Reply:
Moreover, we can merge this PR without publishing any new release. It's a bug-fix...

Reply:
Yes, it's a bugfix. Therefore it should be 0.8.1 instead of 0.9.0

Reply:
> mmmm ok, I didn't know that the v0.8.0 was already published. So, first of all we should tag this version in GitHub's releases

Yes, I forgot to tag it in GitHub 😓

Reply:
jajaja sorry I'd want to say that it's NOT a bugfix, in the end you can work properly with master current dependencies.

Reply:
@gorkalaucirica ping